setup-machine.sh

```diff
@@ -54,18 +54,20 @@ MODEL_NAME=${MODEL_NAMES[$CHOICE_INDEX]}
 echo "Excellent, let's set up ${CHOICE}."
 
 echo
-read -p "Is this image for QA where you want sudo privileges, the ability to record screengrabs, etc.? [y/N] " qa_image_flag
+read -p "Is this image for QA where you want sudo privileges, terminal access via TTY2, the ability to record screengrabs, etc.? [y/N] " qa_image_flag
 
 if [[ $qa_image_flag == 'y' || $qa_image_flag == 'Y' ]]; then
+    IS_QA_IMAGE=1
     VXADMIN_SUDO=1
     ADMIN_PASSWORD='insecure'
-    echo "OK, creating a QA image with vx-admin sudo privileges."
-    echo "Using password insecure for vx-admin user."
+    echo "OK, creating a QA image with sudo privileges for the vx-admin user and terminal access via TTY2."
+    echo "Using password insecure for the vx-admin user."
 else
+    IS_QA_IMAGE=0
     VXADMIN_SUDO=0
     echo "Ok, creating a production image. No sudo privileges for anyone!"
     echo
-    echo "Next, we need to set the admin password for this machine."
+    echo "Next, we need to set a password for the vx-admin user."
```

Comment:
I _think_ as of this change we may actually now be in a place where the vx-admin user doesn't need a password